CONTRIBUTING.md

```diff
@@ -109,6 +109,11 @@ Packages should have uninstaller if auto uninstall doesn't work.
 
 Chocolatey supports PowerShell v2+ because it supports Windows 7+/Windows Server 2008+ (Windows Server 2003 is supported currently, but is deprecated and expected to be removed in a future version). Due to that, PowerShell v2 is expected to work when you run anything here.
 
+### 1.1.13 Chocolatey compatibility
+
+Packages is expected to be compatible with the chocolatey version available 12 months ago.
```

Comment:
Minor nit picks: 

`is` -> `are`
`chocolatey` -> `Chocolatey`

Reply:
I can never remember when to use `is` vs `are`.
Thanks, I'll get those fixed